lib/components/NavMenu.jsx

```diff
@@ -39,13 +39,12 @@ import PropTypes from 'prop-types';
 import Immutable from 'immutable';
 import NavMenuItem from './NavMenuItem';
 
-const NavMenu = ({ menuItems, selectedItemId, onItemSelected, bindHotkey, cssClass }) => (
+const NavMenu = ({ menuItems, selectedItemId, onItemSelected, cssClass }) => (
     <div className={cssClass}>
         {
             menuItems.map((item, index) => {
                 const hotkey = `Alt+${index + 1}`;
                 const onSelected = () => onItemSelected(item.id);
-                bindHotkey(hotkey.toLowerCase(), onSelected);
```

Comment:
Without this line _NavMenuItems_ are not accessible via hotkeys now.

Reply:
As mentioned earlier:

> Only **two** things have hotkeys in core (device/serialport selector, and the "launch application" item from the menu)

Are there any `NavMenuItem` in any application using hotkeys that I'm unaware of?

Reply:
Yes there is. Number 1 app, the BLE.
Also if the created item says in title that it has a hotkey, the user will expect that hotkey to be working.